src/igvfd/schemas/changelogs/mixins.md

```diff
@@ -1,12 +1,10 @@
-## Changelog for mixins.json
+## Changelog for *`mixins.json`*
 
-### Changes since schema version 1
-
-* 3/24/22 Added product_id, lot_id, source mixins
-* 4/5/22 Added additional_description mixin
-* 4/7/22 Added product_id, lot_id, source mixins to a set called product_info
-* 4/27/22 Added collections
-* changed the property *date_created* into *creation_timestamp* and restricted the format to be date-time
-* 6/8/22 Added organism
-* removed mixins for aliases and submitted.  All properies for aliases and submitted have been moved under the basic_item mixin.  All schema objects that include the basic_item mixin will also have the properties previously in aliases and submitted.
-* 6/23/22 Renamed organism to taxa
+* Rename `organism` to `taxa`. (06/23/2022)
```

Comment:
Is it OK that we are including dates in the mixins changelog but not the other changelogs?

Reply:
I think yes, the dates on the mixins are replacing versioning, that is not there as it is not a schema